microsoft/terminal
# Add High Contrast image assets

## Summary of the Pull Request
Add image assets for High Contrast mode

## References
Tagging this issue so it contains a nice list of all the recent HC fixes: #5360

## PR Checklist
* [ ] Closes #6822 
* [X] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #6822

## Detailed Description of the Pull Request / Additional comments
I made several changes to DHowett's script and added it to the repo:
* Add support for generating high contrast icons
* Add the ability to easily edit the "intermediate" (previously "zbase") files for manual hinting
* Appease the spellcheck police

I created new SVGs for HC mode.  There's one SVG for both Black and White modes -- I just invert the colors.  Then I manually hinted the generated bitmaps for the production icons.  I didn't bother hinting the Dev/Pre ones, so the text does get unreadable at small sizes.

## Validation Steps Performed
Your web browser is probably making all these images blurry.  To see how the icon really looks, save the images locally and open in mspaint.

This is the full ramp of icons:
![image](https://user-images.githubusercontent.com/10259764/87476836-e80e9180-c5db-11ea-8c3b-be41521c34c7.png)

Here's the terminal hanging out among a carefully-chosen taskbar full of well-behaved built-in icons:
![image](https://user-images.githubusercontent.com/10259764/87475472-b399d600-c5d9-11ea-9f98-1569b662524c.png)

Same, but at 96dpi:
![image](https://user-images.githubusercontent.com/10259764/87475526-cdd3b400-c5d9-11ea-872c-ac54b2d03f55.png)

A more typical cohort:
![image](https://user-images.githubusercontent.com/10259764/87475724-1ee3a800-c5da-11ea-8cf0-d22c110fd417.png)

Pinned to the Start Menu:
![image](https://user-images.githubusercontent.com/10259764/87475831-4c305600-c5da-11ea-85a9-03b120e730cf.png)
